src/locales/de/translations.json

```diff
@@ -93,29 +93,46 @@
-      },
-      "third": {
-        "title": "UX & UI",
-        "kicker": "Design",
-        "text": "Mit rund 20 Jahren Erfahrung in der Konzeption und Umsetzung digitaler Interfaces bilden wir das gesamte Spektrum der User Experience (UX) ab. Der kleine, sichtbare Teil davon ist das User Interface (UI). Der große, unsichtbare (UX) ist jedoch viel relevanter für das Erreichen ihrer wirtschaftlichen Ziele. Denn nur mit benutzerfreundlichen, leicht verständlichen Journeys über alle Endgeräte, Nutzergruppen und Nutzungskontexte hinweg, erreichen die User und somit Sie ihr Ziel.\nFür unsere Enterprise-Kunden konzipieren und gestalten wir Websites mit modernen, ergonomischen und durchdachten Interfaces, halten uns an UX-Standards und Best Practices, beachten Accessibility Guidelines, testen, messen und optimieren ständig und stellen auf Wunsch alle Komponenten in einem gut strukturierten und erweiterbaren Design System zur Verfügung.\nFor those fields we need:\n· Bulletpoints\n· Bold text like this\n· And text links like this"
-      }
+    "consulting": {
+      "title": "Beratung",
+      "teaser": "Die Erfahrung unserer Mitarbeiter ist unser wertvollster Schatz.",
+      "text": "Die Erfahrung unserer Mitarbeiter ist unser wertvollster Schatz. Wir\n          teilen diesen sehr gerne mit Ihnen und beraten Sie in technischen und\n          konzeptionellen Angelegenheiten.",
+      "list.workshops": "Schulungen & Workshops",
+      "list.analysis": "Technische/UX/UI Analysen & Reports"
+    },
+    "products_services": {
+      "title": "Produkte & Services",
+      "teaser": "Von der Ideation bis zum Rollout. Wir unterstützen Sie.",
+      "text": "Wir unterstützen Sie bei Ihrem nächsten Produkt oder Service mit unseren Experten.",
+      "list.prototyping": "Prototyping",
+      "list.ideation": "Ideation & Proof of Concepts",
+      "list.development": "Product Design & Development"
     },
+    "industries.text": "Derzeit bieten wir unsere Dienstleistungen in den folgenden Branchen an",
+    "industries.title": "Branchen",
+    "industries.bank.title": "Banken",
+    "industries.bank.text": "Umsetzung und Design der gesamten B2C-Plattform.",
+    "industries.automotive.title": "Automotive",
+    "industries.automotive.text": "Unterstützung internationaler Marken im Ausbau von B2B und B2C Plattformen.",
+    "industries.sport.title": "Sport",
+    "industries.sport.text": "Gestaltung & Umsetzung der gesamten B2C Plattform von globalen Playern in der Sportindustrie",
+    "industries.you.title": "Ihre Branche",
+    "industries.you.text": "Wir werden zum Branchenexperten gemeinsamen mit Ihrer Herausforderung. Sprechen Sie uns an.",
+    "industries.insurance.title": "Versicherungen",
+    "industries.insurance.text": "Innovative technische B2B & B2C Plattform für Marktführer, damit diese\n          mit den ständigen Veränderungen in den internationalen Märkten und\n          Regulationen mithalten können.",
```

Comment:
Yesterday I was super frustrated the way we handle translations. The json structure is really not made for human interaction especially with i18n. I simply can't quickly find a key as the names are not full qualified. That's a major pain in the ass and I created a small issue #255 to make note of alternatives.

For a client we have:
1. a plain json file with fully qualified keys
2. we have an extraction tools to build that file from the original sources
3. we use a translation manager which manages the keys, finds missing keys and a big plus: can translate automatically through various service. Instead I did translate by hand, step by step, with deepl..

All that knowledge made it super hard for me to accept the way I have to work here. That's probably the reason I tried out a partially qualified key like this. I'm not proud of providing something that is different than we did before, but it's working and it's much easier to find. So I would like to keep it here as a first example and soon we decide anyway where to put the content and translations for the future 🙏